scripts/import_cldr.py

```diff
@@ -903,6 +904,14 @@ def parse_currency_formats(data, tree):
                         currency_formats[type] = numbers.parse_pattern(pattern)
 
 
+def parse_currency_unit_patterns(data, tree):
+    currency_unit_patterns = data.setdefault('currency_unit_patterns', {})
+    for unit_pattern_elem in tree.findall('.//currencyFormats/unitPattern'):
```

Comment:
This should ignore non-`latn` currency formats at present, just in case there are both `latn` and non-`latn` `currencyFormats` elements within a single locale.

You can (rebase and) use the `_should_skip_number_elem()` function introduced in https://github.com/python-babel/babel/pull/579 to do that.

Reply:
Fixed in 4b5097f6064dda281ce7a1c5ef74e047cfdc558c